File: LogicadeProgramacao/Estrutura de Decisão/2. Java/MediaAluno2.java
import java.util.Scanner;

public class MediaAluno2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Digite a primeira nota: ");
        double nota1 = scanner.nextDouble();
        System.out.print("Digite a segunda nota: ");
        double nota2 = scanner.nextDouble();
        System.out.print("Digite a terceira nota: ");
        double nota3 = scanner.nextDouble();
        
        double media = (nota1 + nota2 + nota3) / 3;
        
        if (media == 10) {
            System.out.println("Aprovado com Distinção - Média " + media);
        } else if (media >= 7) {
            System.out.println("Aprovado - Média " + media);
        } else {
            System.out.println("Reprovado - Média " + media);
        }
    }
}

File: LogicadeProgramacao/Estrutura de Decisão/3. Python/MediaAluno2.py
nota1 = float(input("Digite a primeira nota: "))
nota2 = float(input("Digite a segunda nota: "))
nota3 = float(input("Digite a terceira nota: "))

media = (nota1 + nota2 + nota3) / 3

if media == 10:
    print(f"Aprovado com Distinção - Média {media}")
elif media >= 7:
    print(f"Aprovado - Média {media}")
else:
    print(f"Reprovado - Média {media}")
